src/main/java/org/gbif/utils/file/FileUtils.java

```diff
@@ -660,7 +689,42 @@ public void sort(
       Comparator<String> lineComparator,
       boolean ignoreCase)
       throws IOException {
-    LOG.debug("Sorting " + input.getAbsolutePath() + " as new file " + sorted.getAbsolutePath());
+    sort(Collections.singletonList(input), sorted, encoding, column, columnDelimiter, enclosedBy, newlineDelimiter, ignoreHeaderLines, lineComparator, ignoreCase);
+  }
+
+  /**
+   * Sorts the input file into the output file using the supplied delimited line parameters.
+   *
+   * This method is not reliable when the sort field may contain Unicode codepoints outside the Basic Multilingual Plane,
+   * i.e. above \uFFFF. In that case, the sort order differs from Java's String sort order.  This should not be a problem
+   * for most usage; the Supplementary Multilingual Planes contain ancient scripts, emojis, arrows and so on.
+   *
+   * TODO: This method is globally synchronized, in case multiple sorts are attempted to the same file simultaneously.
+   * This could be improved to allow synchronizing against the destination file, rather than for all sorts.
```

Comment:
I might be overlooking something, but where does this synchronization happen?

Suggest remove the "TODO:"